File: token_selector.py
from config import CONFIG

def select_buy_token(token_address: str):
    CONFIG["BUY_TOKEN_ADDRESS"] = token_address
    print(f"Buy token updated to: {token_address}")

def select_target_token(token_address: str):
    CONFIG["TARGET_TOKEN_ADDRESS"] = token_address
    print(f"Target token updated to: {token_address}")

# For testing:
if __name__ == "__main__":
    select_buy_token("0xNewBuyTokenAddressExample...")
    select_target_token("0xNewTargetTokenAddressExample...")

File: rust module/token_selector.rs
pub mod token_selector {
    use super::config::AppConfig;

    pub fn set_buy_token(config: &mut AppConfig, token_address: &str) {
        config.buy_token_address = token_address.to_string();
        println!("Buy token set to: {}", token_address);
    }

    pub fn set_target_token(config: &mut AppConfig, token_address: &str) {
        config.target_token_address = token_address.to_string();
        println!("Target token set to: {}", token_address);
    }
}
